frontend/app/lib/utils.ts

```diff
@@ -320,6 +326,51 @@ export const getWebMonetizationLink = () => {
   }
 }
 
+export function getDefaultData(): { default: ElementConfigType } {
+  return {
+    default: {
```

Comment:
The `default` key should be responsibility of caller, how they use default data.

```suggestion
export function getDefaultData(): ElementConfigType {
  return {
```